src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package edu.cornell.kfs.rass.batch.service.impl;
+
+import edu.cornell.kfs.rass.batch.service.RassService;
+
+public class RassServiceImpl implements RassService {
```

Comment:
should we have a member variable of this service hold the path to the staging directory, and then set that value in Spring?

Reply:
will add